### PR TITLE
Voice assistant: add Mario Tutorial

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

- add a tutorial on how to reproduce the Mario demo using the OpenAI integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/projects/worlds-most-private-voice-assistant.markdown [hunk 1]
```diff
@@ -71,6 +71,31 @@ your smart home and issue commands and get responses.
 
 <lite-youtube videoid="eLx8_NAqptk" videotitle="Give your voice assistant personality using the OpenAI integration"></lite-youtube>
 
+To reproduce this example, follow these steps:
+
+Note: this procedure requires an OpenAI account. To just run the example, the free trial option is sufficient. No need to leave your credit card information.
+
+1. [Setup an OpenAI account and install the OpenAI conversation](/integrations/openai_conversation/) integration.
```
Comment (frenck) [inline source/projects/worlds-most-private-voice-assistant.markdown]:
```suggestion
1. [Set up an OpenAI account and install the OpenAI conversation](/integrations/openai_conversation/) integration.
```

File: source/projects/worlds-most-private-voice-assistant.markdown [hunk 2]
```diff
@@ -71,6 +71,31 @@ your smart home and issue commands and get responses.
 
 <lite-youtube videoid="eLx8_NAqptk" videotitle="Give your voice assistant personality using the OpenAI integration"></lite-youtube>
 
+To reproduce this example, follow these steps:
+
+Note: this procedure requires an OpenAI account. To just run the example, the free trial option is sufficient. No need to leave your credit card information.
+
+1. [Set up an OpenAI account and install the OpenAI conversation](/integrations/openai_conversation/) integration.
+1. Create a Mario personality.
+   * Once you installed the OpenAI integration, select **Configure**.
```
Comment (frenck) [inline source/projects/worlds-most-private-voice-assistant.markdown]: Just a thought... should this need to explain where to find configure? As in, there is no real pointer to the integrations dashboard here?

Comment (c0ffeeca7) [reply source/projects/worlds-most-private-voice-assistant.markdown]: good point. Thank you!

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**